public_html/style.css

```diff
@@ -0,0 +1,255 @@
+/* Common style css */
+:root {
+
+    /* color */
+    --color-primary: #177d89;
```

Comment:
I believe this is supposed to be `#177e89`

Reply:
I got the code from figma